lib/experimental/html/class-wp-html-tag-processor.php

```diff
@@ -1412,6 +1412,40 @@ public function get_attribute( $name ) {
 		return html_entity_decode( $raw_value );
 	}
 
+	/**
+	 * Returns the names of all attributes matching a given prefix in the currently-opened tag.
```

Comment:
This returns the comparable name, which might be different than the name of the attribute as it was found in the HTML, and which might have a different casing than the given prefix.

It would be helpful to communicate that this returns _the comparable names of all attributes prefixed by a comparable version of the given prefix_ or something to that nature. A couple examples of this mismatch might also be helpful so someone can see in their IDE instantly that they could call `_with_prefix( "DATA-" )` and have it return `["data-id", "data-label"]` on the HTML `<span dAtA-iD=5 dATa-LabEl=blarg>`

Reply:
Sounds good! Yeah, I think I had an earlier iteration of this description containing words like "normalized" or "lowercase", but it ended up sounding a bit too contrived. (I did want to avoid "comparable", since while that's used as a variable name, I only found one instance in a PHPDoc, so it didn't seem like we were widely using it, and I'm not sure it's self-explanatory enough).

Reply:
263cfa13c1bddec3149d6eda9f6957ba90a1ed05